scripts/yarn-install.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
i'm generally open to scripts but i think sometimes bash is a little harder for people and you may run into env issues. also - i've seen this in the past where people do a general git diff to check for changes instead of yarn install, which may catch things like autogenerated changes from types, etc. do you have a preference between checksums and git diffs?

Reply:
now that you mention it, i'm in favor of using `git diff` because it will help us deal with gql codegen issues in the interface repo too.